include/silo/storage/column/indexed_string_column.h

```diff
@@ -0,0 +1,72 @@
+#ifndef SILO_INDEXED_STRING_COLUMN_H
+#define SILO_INDEXED_STRING_COLUMN_H
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <boost/archive/binary_iarchive.hpp>
+#include <boost/archive/binary_oarchive.hpp>
+#include <roaring/roaring.hh>
+
+#include "silo/common/bidirectional_map.h"
+#include "silo/storage/column/column.h"
+
+namespace boost::serialization {
+struct access;
+}
+
+namespace silo::storage::column {
+
+class IndexedStringColumnPartition : public Column {
+   friend class boost::serialization::access;
+
+  private:
+   template <class Archive>
+   [[maybe_unused]] void serialize(Archive& archive, const unsigned int /* version */) {
+      // clang-format off
+      archive & value_ids;
+      archive & indexed_values;
```

Comment:
```suggestion
      archive& value_ids;
      archive& indexed_values;
```
for consistent formatting